GDJS/Runtime/runtimeobject.js

```diff
@@ -217,6 +286,7 @@ gdjs.RuntimeObject.prototype.onDestroyFromScene = function(runtimeScene) {
  * @return {Object} The internal rendered object (PIXI.DisplayObject...)
  */
 gdjs.RuntimeObject.prototype.getRendererObject = function() {
+    return {};
```

Comment:
`return undefined`

Returning an empty object is almost always a bad idea as it creates problem down the road (the function getting it can easily check for undefined/null with a if. If you return an empty object, it thinks that it got a valid object, because in JS an empty object evaluates to true, so it continue and boom, the code crash later in an unrelated function).

Reply:
The exception is of course if the object represents some kind of empty map, but this is not the case here. Either we return a valid PIXI.DisplayObject (or something else depending on the renderer) or we return undefined or null (I would tend to favor `null` in general, but here you can use undefined to be sure to be 100% compatible with the old version).